scripts/python/README.md

```diff
@@ -13,7 +15,13 @@ The init script will have to be run from the context of the current session (in
 immediately make use of the installed environment and installed libraries. 
 Use the following command:
 ```shell
-source init-pyenv.sh
+   source init-pyenv.sh
+```
+
+### direnv
```

Comment:
This is a choice of how you want to handle local environments, no problem to add it here, but then we should also link to the documentation of direnv itself.